installer/netdata-updater.sh

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env bash
+
+# this script will uninstall netdata
+
+# Variables needed by script:
+#  - PATH
+#  - CFLAGS
+#  - NETDATA_CONFIGURE_OPTIONS
+#  - REINSTALL_PWD
+#  - REINSTALL_COMMAND
+
+force=0
+[ "${1}" = "-f" ] && force=1
+
+source installer/.environment.sh || exit 1
+
+# make sure we have a UID
+[ -z "${UID}" ] && UID="$(id -u)"
+INSTALL_UID="${UID}"
+if [ "${INSTALL_UID}" != '${UID}' ]; then
```

Comment:
This condition is always `false`.
`INSTALL_UID` should be stored in `environment.sh`.

Reply:
INSTALL_UID is not used by either uninstaller nor by updater so it shouldn't be dumped to `environment.sh`.

Reply:
there is no variable `INSTALL_UID` in installer script.

Reply:
hm... Still the condition is always `false`.

```
INSTALL_UID="${UID}"
if [ "${INSTALL_UID}" != '${UID}' ]; then ... fi
```

There is no way for `INSTALL_UID` not to be equal to `UID`, since the line above just copied `UID` into `INSTALL_UID`.

Reply:
There is no INSTALL_UID anywhere. You commented an outdated thread.